suites/squid/rgw/tier-2_rgw_ms-archive.yaml

```diff
@@ -248,299 +253,104 @@ tests:
             script-name: user_create.py
             config-file-name: non_tenanted_user.yaml
             copy-user-info-to-site: ceph-arc
+            timeout: 300
       desc: create non-tenanted user
       module: sanity_rgw_multisite.py
       name: create non-tenanted user
       polarion-id: CEPH-83575199
-  - test:
-      name: notify copy events with kafka_broker_persistent
-      desc: notify copy events with kafka_broker_persistent
-      polarion-id: CEPH-83574066
```

Comment:
This tescase loks like missing in newly created files and changes of existing file

Reply:
Its already covered in a multisite suite.